test/functional-tests/pageObjects/createCaseWizardPage.js

```diff
@@ -1,17 +1,15 @@
 BasePage = require('./basePage');
 let FieldUtils = require('../utils/fieldUtils.js');
-
-
+Button = require('./webdriver-components/button.js')
 
 
 class CreateCaseWizardPage extends BasePage{
 
     constructor() {
       super();
-      this.progressButton = 'button[type=submit]';
+      this.progressButton = new Button('button[type=submit]');
```

Comment:
Can this be named as 'continueButton' to avoid confusion with the actual button name please

Reply:
done